README.md

```diff
@@ -28,16 +28,12 @@ To have the best experience when using it with UDFs, [creating a local conda env
 ```bash
 pip install snowflake-snowpark-python
 ```
-Optionally, you need to install pandas in the same environment if you want to use pandas-related features:
-```bash
-pip install "snowflake-snowpark-python[pandas]"
-```
-Optionally, you need to install Modin in the same environment if you want to use Snowpark pandas features:
+To use the [Snowpark pandas API][Snowpark pandas developer guide], you can optionally install the following, which installs [modin][modin] in the same environment. The Snowpark pandas API provides a familiar interface for pandas users to query and process data directly in Snowflake.
 ```bash
 pip install "snowflake-snowpark-python[modin]"
 ```
 
-### Create a session and use the Snowpark Python APIs
+### Create a session and use the Snowpark DataFrame API
```

Comment:
since we are using Snowpark Python API for the other parts of doc, shall we keep this as Snowpark Python API

Reply:
Changing this back to Snowpark Python API.